.github/workflows/pr-benchdiff.yml

```diff
@@ -0,0 +1,138 @@
+        uses: khan/pull-request-comment-trigger@master
+        id: check-comment
+        with:
+          trigger: '/benchdiff'
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+      - name: Set output
+        id: output
+        run: |
+          echo '::set-output name=triggered::${{ github.event_name == 'pull_request' || steps.check-comment.outputs.triggered }}'
+          echo '::set-output name=pr_number::${{ github.event.pull_request.number || github.event.issue.number }}'
+
+  benchdiff:
+    name: Performance regression check
+    needs: [trigger]
+    if: needs.trigger.outputs.triggered == 'true'
+    runs-on: ubuntu-latest
+    env:
+      # In markdown URL syntax
+      RUNS_URL: '[${{ github.workflow }} #${{ github.run_number }}](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})'
+    steps:
+      - name: Report job start
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: '${{ needs.trigger.outputs.pr_number }}'
+          body: |
+            ## Benchdiff
+
+            Runs: ${{ env.RUNS_URL }}
+
+            Job is started.
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Checkout pull request HEAD
+        id: head
+        run: |
+          # Checkout HEAD of pull request and set its sha to step output
+          git fetch origin pull/${{ needs.trigger.outputs.pr_number }}/head:PR_HEAD
+          git checkout PR_HEAD
+          echo "::set-output name=head::$(git rev-parse HEAD)"
+
+          # Set modified packages to step output
+          # git diff: get difference between HEAD and baseline
+          # grep: filter non-go files
+          # xargs -r: --no-run-if-empty, ignore empty line
+          # xargs dirname: keep only the directory name (go packages)
+          # xargs ls: filter non-exist files
+          # sort | uniq: dedup
+          pkgs=$(git diff --name-only ${{ github.sha }} | grep '.go$' | xargs -r dirname | xargs -r ls -d 2>/dev/null | sort | uniq)
+          if [ ! -z "${pkgs}" ]; then
+            # awk: Add "./" prefix to let `go test` known they are relative paths
+            # tr: join paths to one line, otherwise benchdiff cannot recognize it
+            pkgs=$(echo "${pkgs}" | awk '{print "./" $0}' | tr '\n' ' ')
+          fi
+          echo "::set-output name=pkgs::${pkgs}"
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Benchdiff
+        uses: WillAbides/benchdiff-action@v0.3.3
+        id: diff
+        if: steps.head.outputs.pkgs != ''
+        with:
+          benchdiff_version: 0.7.1
+          status_sha: ${{ steps.head.outputs.head }}
+          status_name: Benchdiff result
+          status_on_degraded: neutral
+          benchdiff_args: |
+            --packages="${{ steps.head.outputs.pkgs }}"
+            --count=10
+            --warmup-count=1
+            --benchtime=100000x
```

Comment:
It may not be easy,  `--benchtime` take effects for all `--packages`, and both them are action-level configuration, it is not easy to run a action for every single package.

If we can make sure that we changes at most one package in a PR, we can create a mapping from package name to `--benchtime` (even more bench configuration), thing will be easier.

Reply:
OK, let's use benchtime=1s at least in this PR.